assets/src/dashboard/app/views/myStories/index.js

```diff
@@ -80,6 +81,22 @@ function MyStories() {
     });
   }, [stories, storiesOrderById]);
 
+  const handlePreviewStory = useCallback(
+    (e, story) => {
+      previewVisible.set(e, story);
+    },
+    [previewVisible]
+  );
+
+  if (previewVisible.value) {
```

Comment:
I totally get the pattern here, but the naming of `previewVisible` feels like a boolean where the `value` is the actual story content. It's a minor nit, but unless I'm aware of the logic, I am a bit confused on line `94`.

Reply:
I can agree with this comment.  Maybe it could be called `activePreview`?

Reply:
yeah for sure! i originally had this as just `preview` when this whole thing started, it's been a journey but I agree `activePreview` is a very solid idea.